src/submission/template-api.ts

```diff
@@ -0,0 +1,49 @@
+import * as manager from "../lectern-client/schema-manager";
+import { Request, Response } from "express";
+import { loggerFor } from "../logger";
+import { SchemaDefinition, ValueType } from "../lectern-client/schema-entities";
+import { setStatus, Status } from "../app-health";
+
+const L = loggerFor(__filename);
+
+export const get = async (req: Request, res: Response) => {
+  const schemaName: string = req.params.schemaName;
+  const schemasDictionary = manager.instance().getCurrent();
+  if (!schemasDictionary) {
+    setStatus("schema", { status: Status.ERROR });
+    return res.status(404).send("no schema dictionary found");
+  }
+  const schema = schemasDictionary.schemas.find((schema) => {
+    return schema.name == schemaName;
+  });
+  if (!schema) {
+    setStatus("schema", { status: Status.ERROR });
+    return res.status(404).send("no schema named " + schemaName + " found");
+  }
+  const template = createTemplate(schema);
+  return res.status(200).send(template);
```

Comment:
I believe the gateway needs to get an actual file not a string, can you confirm that with Jon and modify